CMSgov/design-system
# [WNMGDS-1144] storybook for child systems

## Summary
Configuring storybook for child systems

### Added
New commands for starting storybook development for child systems
- `yarn storybook:healthcare` will spin up storybook with all stories from core & healthcare. It will also use healthcare styles
- `yarn storybook:medicare` will spin up storybook with all stories from core & medicare. It will also use medicare styles

Added one child-specific story for each system to test the build.

### Changed
Bumped the .nvmrc node version to the latest version of 14

## How to test
Try out the ☝🏻 commands (make sure to do `nvm use` first!)
I compared the button styles to what I see on the doc site and they looked similar, so I think it's working
